src/liblibc/lib.rs

```diff
@@ -4130,7 +4149,7 @@ pub mod consts {
             pub const _SC_THREAD_THREADS_MAX : c_int = 90;
             pub const _SC_THREADS : c_int = 91;
             pub const _SC_TTY_NAME_MAX : c_int = 107;
-            pub const _SC_ATEXIT_MAX : c_int = 46;
+            pub const _SC_ATEXIT_MAX : c_int = 146;
             pub const _SC_XOPEN_CRYPT : c_int = 117;
```

Comment:
Good catch again!  Thank you for taking the time to look at my patch so closely.  I must have added the '1' accidentally when I was going through the constants.